Resources/doc/overriding_controllers.rst

```diff
@@ -12,9 +12,9 @@ bundle named ``AcmeUserBundle`` that declares itself a child of FOSUserBundle.
 
 .. code-block:: php
 
-    // src/AppBundle/AcmeUserBundle.php
+    // src/Acme/UserBundle/AcmeUserBundle.php
```

Comment:
When we started a new Symfony application `AppBundle` already exists, in this case for create a child user bundle we should use a different namespace IMHO. [Reference (Overriding Default FOSUserBundle Templates)](https://github.com/FriendsOfSymfony/FOSUserBundle/blame/master/Resources/doc/overriding_templates.rst#L110-L122).